File: solvers.python/_common/ini_stats.py
# initialize solver statistics
def ini_stats(prob):
    return {
        'name': prob,   # problem name
        'tm': [],       # timing
        'err': [],      # numerical error
        'flr': [],      # failure
        'shrp': [],     # boundary sharpness
        'k': [],        # number of roots
        'kr': []        # number of real roots
    }

File: solvers.matlab/_common/ini_stats.m
% initialize solver statistics
function stats = ini_stats(prob)

    stats = struct(...
        'name',prob,...    % problem name
        'solver','',...    % problem solver
        'tm',[],...        % timing
        'err',[],...       % numerical error
        'flr',[],...       % failure
        'shrp',[],...      % boundary sharpness
        'k',[],...         % number of roots
        'kr',[]);          % number of real roots

end